sdks/java/core/src/test/java/org/apache/beam/sdk/util/HistogramDataTest.java

```diff
@@ -332,4 +332,19 @@ public void testExponentialBuckets_NumBuckets() {
     HistogramData negativeScaleBucket = HistogramData.exponential(-3, 500);
     assertThat(negativeScaleBucket.getBucketType().getNumBuckets(), equalTo(4));
   }
+
+  @Test
+  public void testStatistics() {
+    HistogramData histogram1 = HistogramData.linear(0, 10, 10);
+    for (int i = 0; i < 10; i++) {
+      histogram1.record(i * 10.0);
+    }
+
+    assertThat(histogram1.getMean(), equalTo(45.0));
+    double sum_of_squared_deviations = 0;
```

Comment:
for java, camel case is preferred https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
done